### ISSUE TITLE
Blind and visually impaired users cannot turn on Talkback from keyboard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#383]
```
What steps will reproduce the problem?
Android platgforms with keyboard support

What is the expected output? What do you see instead?
Blind and visually impaired users should be able to turn on Talkback from the 
keyboard.
This is not possible and requires sighted assistance.

What version of the product are you using? On what operating system?
Many Android platforms with multiple versions of Android and Talkback.

Please provide any additional information below.
If this feature was supported by Android it would be possible for blind and 
visually impaired users to turn on Talkback on platforms that support keyboard 
input.  
Some Android platforms do not have touch screen input so the ability to turn on 
Talkback requires sighted assistance.
NOTE: It is important that this feature is supported on Android first boot so 
that blind and visually impaired users can use the initial setup features of 
Android.


```

Original issue reported on code.google.com by `4RJa...@gmail.com` on 18 Aug 2014 at 1:40

**Comment by GoogleCodeExporter** [comment]
```
I think pressing and holding back would work nicely. This would make it 
possible to start TalkBack on emulators for PC. That's how it is on Android TV, 
so why not? I'm going for consistency here.
```

Original comment by `pbsinn...@gmail.com` on 6 Feb 2015 at 12:15

**Comment by GoogleCodeExporter** [comment]
```
Please add this.  I think triple press of home would work 
```

Original comment by `dennisl1...@gmail.com` on 9 Feb 2015 at 12:02
